src/cli.rs

```diff
@@ -266,6 +270,15 @@ impl Rover {
         }
     }
 
+    pub(crate) fn get_checks_timeout_seconds(&self) -> Result<u64> {
+        // default to 5 minutes
+        Ok(self
+            .get_env_var(RoverEnvKey::ChecksTimeoutSeconds)?
+            .unwrap_or_else(|| "300".to_string())
+            .parse::<u64>()
+            .unwrap())
+    }
```

Comment:
don't want the unwrap here i don't think as it could potentially panic if someone sets the env var to something that is not a valid `u64` - we should attempt to parse the env var first, if the env var is valid use it, if not, return the parse error, it the env var isn't set at all, just use `300` the `u64` not `300` the string

```suggestion
            .get_env_var(RoverEnvKey::ChecksTimeoutSeconds).map(|r| r.parse::<u64>()?)?
            .unwrap_or_else(|| 300)
    }
```